Sources/MapboxMaps/Style/StyleURL.swift

```diff
@@ -0,0 +1,97 @@
+import Foundation
+
+/// Enum representing the latest version of the Mapbox styles (as of publication). In addition,
+/// you can provide a custom URL or earlier version of a Mapbox style by using the `.custom` case.
+public enum StyleURL: Hashable, RawRepresentable {
+    public typealias RawValue = URL
+
+    /// Mapbox Streets is a general-purpose style with detailed road and transit networks.
+    case streets
+
+    /// Mapbox Streets is a general-purpose style tailored to outdoor activities.
+    case outdoors
+
+    /// Mapbox Light is a subtle, light-colored backdrop for data visualizations.
+    case light
+
+    /// Mapbox Dark is a subtle, dark-colored backdrop for data visualizations.
+    case dark
+
+    /// The Mapbox Satellite style is a base-map of high-resolution satellite and aerial imagery.
+    case satellite
+
+    /// The Mapbox Satellite Streets style combines the high-resolution satellite and aerial imagery
+    /// of Mapbox Satellite with unobtrusive labels and translucent roads from Mapbox Streets.
+    case satelliteStreets
+
+    /// The URL to a custom style. The URL may be a full HTTP or HTTPS URL, a Mapbox style URL
+    /// (mapbox://styles/{user}/{style}), or a path to a local file relative to the application’s
+    /// resource path.
+    case custom(url: URL)
+
+    /// Underlying URL for the StyleURL
+    public var url: URL {
+        return rawValue
+    }
```

Comment:
Is this required so as not introduce a breaking change? Would we just want to expose rawValue instead?

Reply:
It was more of a nicety. But let's consider removing.